.vib/moodle/goss/moodle.yaml

```
@@ -0,0 +1,33 @@
+file:
+  /opt/bitnami/apache/conf/vhosts/moodle-vhost.conf:
+    exists: true
+    contains:
+    {{ range $rule := .Vars.apache.rules }}
+      - {{ $rule }}
+    {{ end }}
+  /opt/bitnami/php/etc/php.ini:
+    exists: true
+    contains:
+      # Checking if the php.ini keys and pqsql extension are set
+      - extension = pgsql
+      - /^memory_limit/
+      - /^max_input_vars/
+  /opt/bitnami/apache/conf/vhosts/moodle-https-vhost.conf:
+    exists: true
+    contains:
+    {{ range $rule := .Vars.apache.rules }}
+      - {{ $rule }}
+    {{ end }}
```

For the https vhost, I would additionally check the SSLEngine is enabled.
```suggestion
    contains:
      - "SSLEngine on"
      {{ range $rule := .Vars.apache.rules }}
      - {{ $rule }}
      {{ end }}
```